Podfile

```diff
@@ -40,9 +40,9 @@ target 'WooCommerce' do
   pod 'Gridicons', '~> 1.2.0'
 
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressAuthenticator', '~> 1.43.1'
+  # pod 'WordPressAuthenticator', '~> 2.0.0-beta.1'
   # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :commit => ''
-  # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => ''
+  pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'remove-lottie'
```

Comment:
Of course, before merging, we'll want at least want https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/640 to be merged and shipped as a beta or stable.

I've added the `status: do not merge` label to the PR to make that more visible too and avoid accidental premature merge.

Reply:
Nice. Better than just a comment 😅 Plus gets Peril to help us with it, too.